app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,6 @@
+class IngredientsController < ApplicationController
+   def index
```

Comment:
This is a good controller option for US 3, but you could also consider a nested controller: `chefs/ingredients_controller` since this is a list of ingredients just for 1 specific chef